tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSettingsLoader.java

```diff
@@ -0,0 +1,53 @@
+package nl.tudelft.contextproject.tygron;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class TygronSettingsLoader {
+  Properties config;
+  InputStream input = null;
+
+  String username = "";
+  String password = "";
+
+  /**
+   * Groups can individually decide what username they will fall back on if the
+   * loading or reading of the cfg file fails.
+   */
+  public TygronSettingsLoader(String path) {
+    try {
```

Comment:
Please move the try catch to the TygronSettings so it tries to load data through the settings loader, and if it fails, it should fall back to the default tygron settings.